### DISCUSSION TITLE
What is the service which is installed on client machines?

### DISCUSSION THREAD

**Discussion by lonix1** [rustdesk/rustdesk#5090] [Q&A] [answered]
I'm self hosting and it all works.

I installed the debian client. There seems to be a service which I can toggle via the client.

What is that service? Does it need to be always running?

**Comment by dinger1986** [comment, marked as answer]
the service is for unattended access

you can see the status using `systemctl status rustdesk`

**Reply by lonix1** [reply]
Thanks!

I'm a new rustdesk user, so please forgive the newbie question.

When you say "unattended access", do you mean a scenario where a remote host could take control of that machine (which has the service running), without the user first opening the app?

So assuming I only ever want to use the app upon prior agreement - "hey bob, please load the rustdesk app so I can fix your issue" - then can I leave the service off on all clients?

**Reply by dinger1986** [reply]
Yeah that's correct 👍

**Reply by dinger1986** [reply]
Yeah that's correct 👍
